lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -47,11 +47,14 @@
 // styles together, so that card becomes more of
 // a styled container.
 .beta-card {
-  background-color: $white;
```

Comment:
ahh, added it back in but its on a different line so github doesn't recognize it